packages/taquito/signature.json

```diff
@@ -787,8 +787,7 @@
           "lint-staged": {
             "{src,test}/**/*.ts": [
               "prettier --write",
-              "tslint --fix",
-              "git add"
```

Comment:
in the removal intentional? feel like it'll change behaviour

Reply:
In `signature.json`, no. This file is useless now that we don't sign our packages.
In `package.json` it's redundant after the updates. If we keep this we get: 

```
@taquito/core: ⚠ Some of your tasks use `git add` command. Please remove it from the config since all modifications made by tasks will be automatically added to the git commit index.
```